src/cascadia/TerminalControl/TermControl.h

```diff
@@ -162,7 +162,8 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         Timestamp _lastMouseClick;
         unsigned int _multiClickCounter;
         std::optional<winrt::Windows::Foundation::Point> _lastMouseClickPos;
-        std::optional<winrt::Windows::Foundation::Point> _clickDragStartPos{ std::nullopt };
+        std::optional<winrt::Windows::Foundation::Point> _unfocusedClickPos{ std::nullopt };
+        bool _isClickDragSelection{ false };

```

Comment:
We should initialize these in the constructor or `_InitializeTerminal()` along with the others that are already there.